tests/python/relay/test_op_level5.py

```diff
@@ -565,6 +568,28 @@ def test_run(batch, in_channel, size, out_channel, deformable_groups, groups):
     test_run(2, 4, 16, 4, 4, 1)
 
 
+def test_argsort():
+    def verify_argsort(shape, axis, is_ascend):
+        x = relay.var("x", relay.TensorType(shape, "float32"))
+        z = relay.vision.argsort(x, axis=axis, is_ascend=is_ascend)
```

Comment:
Can we also add `dtype="int32"` in the test case?

Reply:
Maybe in a later pr